src/test-d/types.test-d.ts

```diff
@@ -0,0 +1,48 @@
+import * as React from 'react';
+import { expectAssignable, expectType } from 'tsd';
+import {
+  adminLayerColorsType,
+  ADMN0_PCODE,
+  EmptyObject,
+  FlexComponent,
```

Comment:
Should we use this one istead? https://github.com/onaio/js-tools/blob/master/packages/react-utils/src/types.ts#L7

Reply:
yeah will remove that redundancy